src/postgres/src/test/regress/yb_pg_limit_schedule

```diff
@@ -0,0 +1,16 @@
+# src/test/regress/yb_pg_limit_schedule
+#
+####################################################################################################
+# This includes tests for LIMIT Clause
+####################################################################################################
+# Dependencies for `yb_pg_limit`
+test: yb_pg_int8
+
+
+# Chain of dependencies for `yb_pg_limit`
+test: yb_pg_create_function_1
+test: yb_pg_create_type
+test: yb_pg_create_table
+test: yb_pg_copy
+
+test: yb_pg_limit
```

Comment:
Seems like overkill to create a schedule for effectively a single test: can you move this to `yb_pg_misc_serial_schedule` (unless it has no dependencies: then it goes to `yb_pg_misc_independent_serial_schedule`)?